### ISSUE TITLE
Using trace will trigger metrics to collect a large number of bugs

### ISSUE THREAD

**Issue by Xuzan9396** [openobserve/openobserve#4621]
### Which OpenObserve functionalities are the source of the bug?

metrics

### Is this a regression?

Yes

### Description

Sdk submits a trace, finds that a lot of metrics are generated, and then shuts down the sdk service. Metrics will always generate a lot of metrics regularly. I just submitted a trace span, and my metrics automatically helped me generate 1GB of data a day

![image](https://github.com/user-attachments/assets/666f368e-e02d-42fa-a598-000f33c63e47)

docker
```
version: '3.8'

services:
  openobserve:
    image: openobserve/openobserve:latest
    container_name: openobserve
    ports:
      - "5080:5080"  # OpenObserve Web UI
      - "9200:9200"  # OpenObserve Ingest API
      - "5081:5081"  # OpenObserve gRPC API
    environment:
      ZO_ROOT_USER_EMAIL: "admin@example.com"  # 设置 root 用户的电子邮件
      ZO_ROOT_USER_PASSWORD: "123456"    # 设置 root 用户的密码


```

### Please provide the version you discovered this bug in (check about page for version information)

```true
v0.11.0
7bbe192c015640ce643364b83b8ebb9c76a13331
2024-09-02T07:42:45Z
```

**Comment by Xuzan9396** [comment]
[sample-tracing-golang.zip](https://github.com/user-attachments/files/17096150/sample-tracing-golang.zip)
go-sdk, Once requested, metrics polls to generate data

**Comment by hengfeiyang** [comment]
fixed, in `v0.13.2-rc4`
